### DISCUSSION TITLE
Using bootstrap branding

### DISCUSSION THREAD

**Discussion by thenewguy** [bootstrap-email/bootstrap-email#100] [General]
I am trying to wrap my head around what is required to achieve a consistent brand look for bootstrap based web site and bootstrap based emails.

If a brand site is current styled using scss bootstrap, does one take that theme and it works out of the box with bootstrap email?  I.e. I just take the bootstrap theme scss files from the website and plug it into the bootstrap-email compiler somehow and then get a branded email?

Or does a designer generate a website bootstrap theme from https://github.com/twbs/bootstrap/tree/main/scss and also a completely separate email bootstrap theme from https://github.com/bootstrap-email/bootstrap-email/tree/master/core/scss to create a similar look and feel?

**Comment by stuyam** [comment]
You would want to use something like the bootstrap email config file to load custom scss to customize things like colors etc and could store than in a central place. Note this would require you to use the 1.0 alpha. More information about customizing can be found here: https://v1.bootstrapemail.com/docs/configure

**Reply by thenewguy** [reply]
Hey! Thanks for the reply. Still a little confused as to how these customizations relate to customizations already created for bootstrap web styles.

So are these General Variables / Color Variables / Utility Variables the same as used by bootstrap for the web?  For example, if I have a [custom variables.scss](https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss) already, does that apply to the bootstrap for email theme.  Or is the "bootstrap for email" css package completely separate?

**Reply by stuyam** [reply]
The Bootstrap Email package is completely separate from regular Bootstrap. However it does use a lot of the same SASS variable names and such. The docs were recently update so check out this page again: https://v1.bootstrapemail.com/docs/configure

But essentially you should be able to make a file `bootstrap-email.scss` in the root of your project (or you can customize the location). You can see the SASS variable files you can override: https://v1.bootstrapemail.com/docs/configure#customize-sass
```scss
@import 'path/to/your/custom/sass';
$primary = $primary-color; // example of how to map variable to different name if need be.

//= @import bootstrap-email;
```

To import a file like in the example `'path/to/your/custom/sass'`, you need to add a sass load path via config or a config file. Something like this:
```ruby
BootstrapEmail.configure do |config|
  # Array of folders to add to the SASS load path to support imports in the custom SASS files
  config.sass_load_paths = [File.expand_path('path/to/your/custom/sass', __dir__)]
end
```
